### ISSUE TITLE
Pagination appears broken for package tags

### ISSUE THREAD

**Issue by mikelove** [Bioconductor/support.bioconductor.org#85]
https://support.bioconductor.org/tag/DESeq2/

![Screen Shot 2022-06-15 at 5 05 27 PM](https://user-images.githubusercontent.com/69203/173929794-db37adf2-111d-4217-9c2c-b0b6690a605d.png)

There is no next page link.

**Comment by mikelove** [comment]
I think this is still the case?

https://support.bioconductor.org/tag/limma/

1 page only for all time limma questions.

**Comment by lshep** [comment]
Yes I don't think we got to fixing this yet.  We can look into adding pagination back shortly.

**Comment by mikelove** [comment]
I noticed something. This is also a bug on biostars (the tag-level paginations buttons aren't hyperlinked to anything). And adding `?page=2` to the URL does nothing to the page.

**Comment by mikelove** [comment]
Just bumping this. I know there is likely limited availability of folks to tackle this and it's upstream of Bioc which also means harder to fix...

**Comment by lshep** [comment]
fixed in 36673edac039f9b1cf50d844a054789e99c729ce

**Comment by mikelove** [comment]
Thanks @lshep !!
